CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+Notable changes to this project are documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+Breaking changes:
+
+New features:
+
+Bugfixes:
+
+Other improvements:
+
+## [v1.2.2](https://github.com/purescript-web/purescript-web-dom-xpath/releases/tag/v1.2.2) - 2019-08-18
+
+Raised upper bound for `purescript-web-dom`
+
```

Comment:
This is particularly sparse. We probably don't need to transcribe all 7 tags here, but we could at least add these:

```
# [v1.2.1](https://github.com/purescript-web/purescript-web-dom-xpath/releases/tag/v1.2.0) - 2019-06-26

- Added tests and notes pertaining to `xs:boolean` vs `boolean_type`
- Removed dependency on `web-events`
- Updated for compatibility with PureScript 0.13

# [v1.2.0](https://github.com/purescript-web/purescript-web-dom-xpath/releases/tag/v1.2.0) - 2019-04-23

- Added typed convenience functions wherever possible.

# [v1.1.0](https://github.com/purescript-web/purescript-web-dom-xpath/releases/tag/v1.1.0) - 2019-04-23

- Added `snapshot` function for returning an `Array` of results.

# [v1.0.0](https://github.com/purescript-web/purescript-web-dom-xpath/releases/tag/v1.0.0) - 2019-04-19

- Initial release of the `web-dom-xpath` library.
```

Reply:
This could be copy/pasted into the CHANGELOG. Most libraries don't need this because they have GitHub releases for the various tags, but in this case the changelog needs a little extra work on the content.

Reply:
Done.